pytest_tests/testsuites/failovers/test_failover_service_kill.py

```diff
@@ -0,0 +1,334 @@
+import logging
+import random
+
+import allure
+import pytest
+from cluster import HTTPGate, MorphChain, StorageNode
+from epoch import ensure_fresh_epoch, wait_for_epochs_align
+from file_helper import generate_file
+from neofs_testlib.hosting import Host
+from neofs_testlib.shell import Shell
+from python_keywords.container import create_container
+from python_keywords.http_gate import (
+    get_object_and_verify_hashes,
+    get_via_http_curl,
+    upload_via_http_gate_curl,
+)
+from python_keywords.neofs_verbs import delete_object, put_object_to_random_node, search_object
+from python_keywords.node_management import check_node_in_map, storage_node_healthcheck
+from tenacity import retry, stop_after_attempt, wait_fixed
+from test_control import wait_for_success
+from wallet import WalletFactory, WalletFile
+from wellknown_acl import PUBLIC_ACL
+
+from steps.cluster_test_base import ClusterTestBase
+
+logger = logging.getLogger("NeoLogger")
+
+NEOFS_IR = "neofs-ir"
+NEOFS_HTTP = "neofs-http"
+NEOFS_NODE = "neofs-storage"
+NEO_GO = "neo-go"
+
+
+@pytest.mark.failover
+@pytest.mark.failover_network
+class TestFailoverServiceKill(ClusterTestBase):
+    PLACEMENT_RULE = "REP 1 IN X CBF 1 SELECT 1 FROM * AS X"
+
+    @pytest.fixture(scope="class")
+    def user_wallet(self, wallet_factory: WalletFactory) -> WalletFile:
+        return wallet_factory.create_wallet()
+
+    @pytest.fixture(scope="class")
+    @allure.title("Create container")
+    def user_container(self, user_wallet: WalletFile):
+        return create_container(
+            wallet=user_wallet.path,
+            shell=self.shell,
+            endpoint=self.cluster.default_rpc_endpoint,
+            rule=self.PLACEMENT_RULE,
+            basic_acl=PUBLIC_ACL,
+        )
+
+    @allure.title("Check and return  status of given service")
+    def service_status(self, service: str, shell: Shell) -> str:
+        return shell.exec(f"sudo systemctl is-active {service}").stdout.rstrip()
+
+    @allure.step("Run health check to node")
+    def health_check(self, node: StorageNode):
```

Comment:
We have similar functionality here:
https://github.com/TrueCloudLab/frostfs-testcases/blob/6e4558d7920415717246409568add971b146ce1b/pytest_tests/testsuites/network/test_node_management.py#L435,L465